lib/eslint-tree.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var path = require('path');
+var ESLintTree = require('broccoli-lint-eslint');
+var escape = require('js-string-escape');
+
+module.exports = function(tree) {
+  var esLintTree = new ESLintTree(tree, {
+    testGenerator: testGenerator,
+  });
+
+  esLintTree.targetExtension = 'eslint.js';
```

Comment:
It looks like this line was causing the issues. If the default `lint-test.js` extension is used instead it works properly...